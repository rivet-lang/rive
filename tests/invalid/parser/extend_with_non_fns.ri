extend u8 {
    const BAD: i32 = 1;

    struct Bad2 {}
}
