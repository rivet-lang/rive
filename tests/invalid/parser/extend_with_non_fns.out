tests/failing/parser/extend_with_non_fns.ri:2:11: error: expected associated function or method
    |
  2 |     const BAD: i32 = 1;
    |           ^
tests/failing/parser/extend_with_non_fns.ri:4:12: error: expected associated function or method
    |
  4 |     struct Bad2 {}
    |            ^
rivetc: error: could not compile package `core`, aborting due to 2 previous errors